docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -382,6 +389,15 @@ public FormValidation doCheckVolumesFromString(@QueryParameter String volumesFro
             return FormValidation.ok();
         }
 
+        @Restricted(DoNotUse.class)
```

Comment:
No one needs to refer to the symbol for now, it is exposed for jelly consumption only. It is not a part of plugin API so it can be changed freely in the future.